layer_factory/assistant_layer/exclusive_qfi.h

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2015-2018 Valve Corporation
```

Comment:
Since you just wrote this, shouldn't the copyright just be 2018?

Reply:
Heh, I wrote the copyright message in 2015.